From 59fb035deebfc7599ac80954938338ac861a8f30 Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Fri, 10 Jun 2016 20:37:48 -0400
Subject: [PATCH 24/58] ti_config_fragments/connectivity.cfg: enable rapidio on
 k2hk

This patch enables support of RapidIO on Keystone2 K2HK
platform.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 ti_config_fragments/connectivity.cfg | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/ti_config_fragments/connectivity.cfg b/ti_config_fragments/connectivity.cfg
index 53f284f..1fe1560 100644
--- a/ti_config_fragments/connectivity.cfg
+++ b/ti_config_fragments/connectivity.cfg
@@ -328,3 +328,13 @@ CONFIG_KEYBOARD_MATRIX=m
 # Rotary Encoder
 CONFIG_INPUT_GPIO_ROTARY_ENCODER=m
 CONFIG_INPUT_GPIO_ROTARY_ENCODER_POLL_MODE_SUPPORT=y
+
+#RAPIDIO
+CONFIG_HAS_RAPIDIO=y
+CONFIG_RAPIDIO=y
+CONFIG_RAPIDIO_MPORT_CDEV=y
+CONFIG_TI_KEYSTONE_RAPIDIO=y
+CONFIG_RAPIDIO_ENUM_BASIC=y
+CONFIG_RIONET=y
+CONFIG_RAPIDIO_ENABLE_RX_TX_PORTS=y
+CONFIG_RAPIDIO_DMA_ENGINE=y
-- 
1.9.1

